lib/Runtime/PlatformAgnostic/Platform/Common/HiResTimer.cpp

```diff
@@ -15,7 +15,7 @@ namespace DateTime
     // This method is expected to return UTC time (See MSDN GetSystemTime)
     inline static double GetSystemTimeREAL()
     {
-#ifndef __APPLE__
+#ifdef HAVE_WORKING_CLOCK_GETTIME
```

Comment:
how does this macro get defined?

Reply:
During pre-build by configure.cmake. It tests the feature on the system and then enables / disables for the build

Reply:
It's part of the PAL cmake I think: https://github.com/Microsoft/ChakraCore/blob/97fdeb81f981c71bc3fc6263b310a810eb5a6374/pal/src/configure.cmake#L352